src/services/ContributingDeveloperAuditService/ContributingDeveloperAuditService.ts

```diff
@@ -0,0 +1,87 @@
+import SOOSHooksApiClient, { IContributorAuditModel } from "../../api/SOOSHooksApiClient";
+import { SOOS_CONSTANTS } from "../../constants";
+import { ScmType } from "../../enums";
+import { soosLogger } from "../../logging";
+import GitHubAudit from "./scm/GitHub/GitHubAudit";
+import FileSystem from "fs";
+import * as Path from "path";
+import { ParamUtilities } from "./utilities";
+
+export interface IContributingDeveloperAuditProvider {
+  audit(implementationParams: Record<string, string | number>): Promise<IContributorAuditModel>;
+  validateParams(implementationParams: Record<string, string | number>): void;
+}
+
+class ContributingDeveloperAuditService {
+  private auditProvider: IContributingDeveloperAuditProvider;
+  public soosHooksApiClient: SOOSHooksApiClient;
```

Comment:
make the services consistent on naming
```suggestion
  public hooksApiClient: SOOSHooksApiClient;
```

Reply:
yeah, i figured since it was inside the service itself clarify in the variable that it's our own hook api since it could potentially be others